packages/capacitor/src/hooks/useOrientation.ts

```diff
@@ -0,0 +1,34 @@
+import { useCallback, useEffect, useMemo, useState } from 'react'
+
+type Orientation = 'portrait' | 'landscape'
+
+function useOrientation() {
+  const [orientation, setOrientation] = useState<Orientation>(
+    window.innerHeight > window.innerWidth ? 'portrait' : 'landscape'
```

Comment:
Could this be based on `portraitQuery.matches`?

Reply:
By the way, also check out [screen.orientation](https://developer.mozilla.org/en-US/docs/Web/API/Screen/orientation) and [this example hook](https://github.com/streamich/react-use/blob/325f5bd69904346788ea981ec18bfc7397c611df/src/useOrientation.ts#L14)

Reply:
REQ: I would prefer to use actual orientation events for this too. this is too simple/brittle